src/els_execute_command_provider.erl

```diff
@@ -48,3 +50,24 @@ execute_command(Command, Arguments) ->
   lager:info("Unsupported command: [Command=~p] [Arguments=~p]"
             , [Command, Arguments]),
   [].
+
+
+%% @doc Strip a server-unique prefix from a command.
+-spec strip_server_prefix(binary()) -> binary().
+strip_server_prefix(PrefixedCommand) ->
+  [_, Command] = binary:split(PrefixedCommand, <<":">>),
+  Command.
+
+%% @doc Add a server-unique prefix to a command.
+-spec add_server_prefix(binary()) -> binary().
+add_server_prefix(Command) ->
+  Prefix = server_prefix(),
+  <<Prefix/binary, ":", Command/binary>>.
+
+%% @doc Generate a prefix unique to this running erlang_ls server.  This is
```

Comment:
Nits: typo `vscdoe`. Also, I would avoid mentioning specific clients in the code/comment and keep the comments editor-agnostic.

Reply:
Ok. In this case it *is* driven by a particular design decision in a particular client